front-end-side/src/Pages/Friends.css

```diff
@@ -0,0 +1,3 @@
+img{
+    height: 300px;
```

Comment:
I guess it won't be really responsive on the phone by using px

Reply:
Oh no